.travis.yml

```diff
@@ -20,10 +21,15 @@ jobs:
         - make install
         - pre-commit autoupdate
         - pre-commit run --all-files
-    - stage: test
+    - stage: unit-test
       script:
         - make install
-        - make test
+        - make unit_test
+    - stage: integration-test
+      if: branch = master
```

Comment:
this is the test on post-merge  scenario.  is that what you want?  or did you want pre-merge testing?

Reply:
I'm not 100% sure what you mean. For clarification: the section `if: branch = master` also resolves to True in a pull request where the base branch (the branch being merged into) is master.

This is the behavior makes sense to me. Do you think there is something wrong with it?

Reply:
i'm not opposed to this and i don't think there's any problem with it.  I was just wondering how you intended this to work to see if it meets your expectations.

This change does not hold up an approval